src/v/cloud_roles/apply_abs_oauth_credentials.h

```diff
@@ -11,6 +11,7 @@
 #pragma once
 
 #include "cloud_roles/apply_credentials.h"
+#include "cloud_roles/signature.h"
```

Comment:
FYI with the new includes work since `signature.h` is a private header to this module it needs to be a local includes (not rooted from `v`. I will followup a PR